packages/react-instantsearch-core-next/src/InstantSearch.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { IndexContext } from './IndexContext';
+import { InstantSearchContext } from './InstantSearchContext';
+import { useInstantSearch } from './useInstantSearch';
+
+import type { UseInstantSearchProps } from './useInstantSearch';
+
+type InstantSearchProps = UseInstantSearchProps & {
+  children?: React.ReactNode;
+};
+
+export function InstantSearch({ children, ...props }: InstantSearchProps) {
+  const search = useInstantSearch(props);
```

Comment:
do we have a use case for useInstantSearch and useIndex outside of this widget?

Reply:
ah for useInstantSearch it could be for refreshing

Reply:
I don't think so. Users would use `useInstantSearchContext` and `useIndexContext` (not exported yet).

For refreshing, I think we can expose its own hook like `const { refresh } = useInstantSearchCache()`.